parade/ddl.sql

```diff
@@ -0,0 +1,160 @@
+CREATE EXTENSION pgcrypto;
```

Comment:
I remember something about extention and postgres that will require to check and install once on public

Reply:
Not sure what you are requesting.  Up until now I've not gone through the up/down scripts, so it was a skipped error.

Now I've changed it to "if not exists", so it should always succeed.  `pgcrypto` is a standard extension, it should always be there.  And it _is_ there on RDS